### PR TITLE
S2H additions

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/filters/filter_base.rb [hunk 1]
```diff
@@ -649,7 +649,8 @@ def cohort_columns_for_select
 
     def cohort_columns_for_dates
       GrdaWarehouse::Cohort.available_columns.select do |column|
-        column.class.ancestors.include?(CohortColumns::CohortDate)
+        # Ignore non-dates and untranslated custom dates
+        column.class.ancestors.include?(CohortColumns::CohortDate) && ! column.title.starts_with?('User')
```
Comment (awisema) [inline app/models/filters/filter_base.rb]: Maybe
```suggestion
        column.class.ancestors.include?(CohortColumns::CohortDate) && ! column.title.match(/^User Date [0-9]+$/))
```
to reduce the chance of accidental hiding?

File: drivers/boston_reports/app/models/boston_reports/street_to_home.rb [hunk 2]
```diff
@@ -361,46 +379,46 @@ def housed_by_cohort
...
-            overall_for_dates[month_start] += active_count
-          end
-          data['columns'] << row
-          data['colors'][cohort] = config["breakdown_1_color_#{i}"]
-        end
-        data['columns'] << ['Total'] + overall_for_dates.values
-        data['colors']['Total'] = config['total_color']
-      end
-    end
+    # def matched_by_cohort
```
Comment (awisema) [inline drivers/boston_reports/app/models/boston_reports/street_to_home.rb:382]: Can this be removed?

**Review by awisema — APPROVED** [review]

File: app/models/filters/filter_base.rb (hunk 1 above)
Comment (eanders) [reply app/models/filters/filter_base.rb]: Thanks.  Using `.match?(/^User Date \d+$/)`

File: drivers/boston_reports/app/models/boston_reports/street_to_home.rb (hunk 2 above)
Comment (eanders) [reply drivers/boston_reports/app/models/boston_reports/street_to_home.rb:382]: Yes, but I'm not 100% convinced it won't come back in the next iteration, so leaving for now.

**Pull request merged**